### PR TITLE
frogatto: unstable-2018-12-18 -> unstable-2021-05-24, unbreak

### PR DESCRIPTION
**Pull request by astro**
###### Motivation for this change

Update. Unbreaks the build.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Added a release notes entry if the change is major or breaking
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by SuperSandro2000 — APPROVED** [review]
diff LGTM

File: pkgs/games/frogatto/data.nix [hunk 1]
```diff
@@ -2,14 +2,14 @@
 
 stdenv.mkDerivation {
   pname = "frogatto-data";
-  version = "unstable-2018-12-18";
+  version = "unstable-2021-05-24";
 
   src = fetchFromGitHub {
     owner = "frogatto";
     repo = "frogatto";
     # master branch as of 2020-12-17
```
Comment (SuperSandro2000) [inline pkgs/games/frogatto/data.nix:10]:
```suggestion
```

**Comment by SuperSandro2000** [comment]
I think we also need to remove https://github.com/NixOS/nixpkgs/blob/master/pkgs/games/frogatto/default.nix#L37.

**Comment by astro** [comment]
Ok, fixed

**Pull request merged**